content/efcore/getting-started/samples/defining-unique-identifiers/composite-key/README.md

```diff
@@ -0,0 +1,10 @@
+# Composite Key
+
+This code example demonstrates how to configure a composite key in EF Core via Fluent API.
+
+## Code Files:
+- [Program.cs](Program.cs)
+- [LibraryContext.cs](LibraryContext.cs)
+- [CheckoutRecord.cs](CheckoutRecord.cs)
+- [Book.cs](../../shared/Book.cs)
+- [Reader.cs](../../shared/Reader.cs)
```

Comment:
This implementation of `Reader` requires `Address`. Use an implementation that doesn't need `Address`